utilities/resource_sync/sync.py

```diff
@@ -548,65 +506,55 @@ def synchronize_table(table, mapping):
-
-    # Copy database configuration
+def process_job(job, mapping):
+    job = organize_job_param(job, mapping)
+    job_name = job['Name']
     try:
-        logger.debug(f"Checking if database '{database_name}' exists in the destination account.")
-        current_database = dst_glue.get_database(Name=database_name)
-        logger.debug(f"Current database '{database_name}' configuration: {current_database}")
-        if args.overwrite_databases:
-            database_argument['Name'] = database_name
-            logger.debug(f"Updating database '{database_name}' with configuration: '{database_argument}'")
+        logger.debug(f"Checking if job '{job_name}' exists in the destination account.")
+        dst_glue.get_job(JobName=job_name)
+        if args.overwrite_jobs:
+            del job['Name']
+            job_update = {'JobName': job_name, 'JobUpdate': job}
+            logger.debug(f"Updating job '{job_name}' with configuration: '{json.dumps(job_update, indent=4, default=str)}'")
             if do_update:
-                dst_glue.update_database(**database_argument)
-            logger.info(f"The database '{database_name}' has been overwritten.")
+                dst_glue.update_job(**job_update)
+            logger.info(f"The job '{job_name}' has been overwritten.")
     except dst_glue.exceptions.EntityNotFoundException:
-        logger.debug(f"Creating database '{database_name}' with configuration: '{database_argument}'")
-        database_argument['DatabaseInput']['Name'] = database_name
+        logger.debug(f"Creating job '{job_name}' with configuration: '{json.dumps(job, indent=4, default=str)}'")
         if do_update:
-            dst_glue.create_database(**database_argument)
-        logger.info(f"New database '{database_name}' has been created.")
+            dst_glue.create_job(**job)
+        logger.info(f"New job '{job_name}' has been created.")
     except Exception as e:
-        logger.error(f"Error occurred in copying database: '{database_name}'")
+        logger.error(f"Error occurred in deploying job: '{job_name}'")
         if args.skip_errors:
             logger.error(f"Skipping error: {e}", exc_info=True)
         else:
             raise
 
-    # Iterate tables under the database
-    if args.src_table_names is not None:
-        logger.debug(f"Sync target table: {args.src_table_names}")
-        table_names = args.src_table_names.split(',')
-        for table_name in table_names:
-            table = src_glue.get_table(DatabaseName=database_name, Name=table_name)
-            synchronize_table(table['Table'], mapping)
-    else:
-        tables = []
-        get_tables_paginator = src_glue.get_paginator('get_tables')
-        for page in get_tables_paginator.paginate(DatabaseName=database_name):
-            tables.extend(page['TableList'])
 
-        for t in tables:
-            synchronize_table(t, mapping)
+def process_table(table, mapping):
```

Comment:
It seems we are removing capability to synchronize database and table partitions. Can you port them? We also need to serialize databases and table partitions in addition to tables.